test/new-e2e/scenarios/ndm/main.go

```diff
@@ -1,34 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
```

Comment:
Why this file was deleted?

Reply:
Because we don't need it. I created it at the beginning to have trigger an environment using `pulumi up`, but it is not required anymore

Reply:
Will change the commit history to better explain it.

Reply:
Added a note https://github.com/DataDog/datadog-agent/pull/19810/files#r1340284138